ROLES.md

```diff
@@ -241,76 +202,33 @@ are responsible for the overall technical health of their functional area.
 
 ### Requirements
 
-Getting to be a tech lead of an existing working group:
-
-- Recognized as having expertise in the group’s subject matter.
-
-- Approver for a relevant part of the codebase for at least 3 months.
-
-- Member for at least 6 months.
-
-- Should have reviewed or contributed to substantial PRs to the codebase.
-
-- Should be a `reviewer` in the OWNERS file.
-
-- Sponsored by the technical oversight committee.
-
-Additional requirements for leads of a new working group:
-
-- Originally authored or contributed major functionality to the group's area.
-
-### Responsibilities and privileges
-
-The following apply to the area / component for which one would be an owner.
-
-- Design/proposal approval authority over the area / component, though
-  escalation to the technical oversight committee is possible.
-
-- Technical review of [feature tracks](./mechanics/FEATURE-TRACKS.md).
-
-- Perform issue triage on GitHub.
-
-- Apply/remove/create/delete GitHub labels and milestones.
-
-- Write access to repo (assign issues/PRs, add/remove labels and milestones,
-  edit issues and PRs, edit wiki, create/delete labels and milestones).
-
-- Capable of directly applying lgtm + approve labels for any PR.
-
-  - Expected to respect OWNERS files approvals and use
-    [standard procedure for merging code](./REVIEWING.md#merging-prs).
-
-- Expected to work to holistically maintain the health of the project through:
+For existing working groups:
 
-  - Reviewing PRs.
+- Recognized as having expertise in the group’s subject matter
 
-  - Fixing bugs.
+- Approver for a relevant part of the codebase for at least 3 months
 
-  - Identifying needed enhancements / areas for improvement / etc.
+- Member for at least 6 months
 
-  - Execute pay-down of technical debt.
+- Should have reviewed or contributed to substantial PRs to the codebase
 
-- Mentoring and guiding approvers, members, and new contributors.
+- Should be an `approver` in the OWNERS file
 
-## Working Group Execution Lead
+- Sponsored by the Technical Oversight Committee
 
-Working group execution leads, or just ‘execution leads’, are responsible for
-the overall health and execution of the working group itself. Execution leads
-work with tech leads to ensure that the working group is making progress toward
-its goals, is aligned with the project roadmap, etc. The execution lead may also
-be the tech lead in a smaller working group, but they are distinct roles.
+- Participant in the working group for at least 3 months, for example as a
```

Comment:
Again, I'd remove the time-frame, but "recognized in the working group as an active participant" seems reasonable.